api/controllers/v1/massif/find.js

```diff
@@ -31,6 +33,19 @@ module.exports = async (req, res) => {
     await MassifService.setNetworks(massif);
     await NameService.setNames(massif.networks, 'cave');
 
+    // complete documents descriptions
+    if (massif.documents && massif.documents.length > 0) {
+      const promisesArray = [];
+      for (let i = 0; i < massif.documents.length; i += 1) {
```

Comment:
Tu peux remplacer par `for (const document of massif.documents)` si tu ne te sers de i que pour accéder au tableau.